pkg/reconciler/knativeeventing/common/replicasenvvarstransform.go

```diff
@@ -73,3 +99,12 @@ func ReplicasEnvVarsTransform(client unstructuredGetter) mf.Transformer {
 		return nil
 	}
 }
+
+func nameExistsContainers(name string, containers []corev1.Container) (bool, int) {
```

Comment:
I'd make this a `findContainer(name string, containers []corev1.Container) *corev1.Container` to give it a more speaking name and avoid the index shenanigans.

Reply:
I changed the name and the way to implement.